.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "cjs": {
+      "presets": ["es2015", "react", "stage-2"],
```

Comment:
ive used `return <WrappedComponent {...props} />;` in the `render` function, which ofc could be avoided if u wish